client/src/routes/GlobalRouter.jsx

```diff
@@ -10,25 +10,35 @@ import { LandingPage } from "../pages/LandingPage/LandingPage";
 import { Login } from "../pages/auth/Login/Login";
 import TopBar from "../components/TopBarApp/TopBarApp";
 import { Chat } from "../pages/Chat/Chat";
+import { InfoUser } from "../pages/InfoUser/InfoUser";
 import { CreateTrip } from "../pages/CreateTrip/CreateTrip";
+import { KompitrailContext } from "../../context/KompitrailContext";
 
 export const GlobalRouter = () => {
+  const { user, token } = useContext(KompitrailContext);
+
   return (
     <BrowserRouter>
-      <TopBar />
-      <NavBarApp>
+      {token && user && <TopBar />}
+      {token && user && (
+        <NavBarApp>
+          <Routes>
+            <Route path={RoutesString.home} element={<Home />} />
+            <Route path={RoutesString.search} element={<Search />} />
+            <Route path={RoutesString.createtrip} element={<CreateTrip />} />
+            <Route path={RoutesString.chat} element={<Chat />} />
+            <Route path={RoutesString.profile} element={<Profile />} />
+            <Route path={RoutesString.infouser} element={<InfoUser />} />
+          </Routes>
+        </NavBarApp>
+      )}
+      {!token && !user && (
```

Comment:
```suggestion
      {(!token || !user) && (
```
¿Podemos revisar esta sugerencia en como tenemos implementado esto?